be/src/olap/row_cursor.cpp

```diff
@@ -28,19 +28,6 @@ using std::string;
 using std::vector;
 
 namespace doris {
-RowCursor::RowCursor()
-        : _fixed_len(0), _variable_len(0), _string_field_count(0), _long_text_buf(nullptr) {}
-
-RowCursor::~RowCursor() {
-    delete[] _owned_fixed_buf;
-    delete[] _variable_buf;
-    if (_string_field_count > 0 && _long_text_buf != nullptr) {
-        for (int i = 0; i < _string_field_count; ++i) {
-            free(_long_text_buf[i]);
```

Comment:
the long_text_buf should release one by one. because string type will realloc the mem

Reply:
i see, realloc in agg_update